sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze.py

```diff
@@ -15,7 +14,7 @@
 try:
     from unittest import mock
 except ImportError:  # python < 3.3
-    import mock  # type: ignore
+    from unittest import mock  # type: ignore
```

Comment:
This will need fixing 😆

Reply:
It's fair that it's too much complicated for pyupgrade :). But at least it's not wrong, just redondant